src/modules/commander/HealthAndArmingChecks/Common.cpp

```diff
@@ -317,3 +317,12 @@ bool Report::report(bool is_armed, bool force)
 			       current_results.health.error, current_results.health.warning);
 	return true;
 }
+
+bool Report::reportIfUnreportedDifferences(bool is_armed)
+{
+	if (_had_unreported_difference) {
+		return report(is_armed, true);
```

Comment:
Mildly related but where's `is_armed` even used in report()? 🤔

Reply:
You are right, it's not used. I removed it in the 2. commit.